Specs/Core/WallGeometrySpec.js

```diff
@@ -82,6 +82,53 @@ describe("Core/WallGeometry", function () {
     expect(geometry).toBeUndefined();
   });
 
+  it("does not create when minimumHeights and maximumHeights are the same", function () {
+    var geometry = WallGeometry.createGeometry(
+      new WallGeometry({
+        positions: Cartesian3.fromDegreesArrayHeights([
+          -115.0,
+          44.0,
+          100000.0,
+          -90.0,
+          44.0,
+          200000.0,
+          -30.0,
+          44.0,
+          200000.0,
+        ]),
+        maximumHeights: [60000.0, 60000.0, 50000.0],
+        minimumHeights: [60000.0, 60000.0, 50000.0],
+      })
+    );
+
+    expect(geometry).toBeUndefined();
+  });
+
+  it("does not create when removing duplicated position results in minimumHeights and maximumHeights are the same", function () {
+    var geometry = WallGeometry.createGeometry(
+      new WallGeometry({
+        positions: Cartesian3.fromDegreesArrayHeights([
+          -115.0,
+          44.0,
+          200000.0,
+          -115.0,
+          44.0,
+          100000.0,
+          -90.0,
+          44.0,
+          200000.0,
+          -30.0,
+          44.0,
+          200000.0,
+        ]),
+        maximumHeights: [40000.0, 60000.0, 60000.0, 50000.0],
+        minimumHeights: [60000.0, 60000.0, 60000.0, 50000.0],
+      })
```

Comment:
I'm not sure if maximumHeights should always be larger than minimumHeights. This seems to work in the master, so I assume I should test as well